rwengine/src/ai/TrafficDirector.cpp

```diff
@@ -85,8 +85,10 @@ std::vector<GameObject*> TrafficDirector::populateNearby(const ViewCamera& camer
 	float halfRadius2 = std::pow(radius/ 2.f, 2.f);
 
 	// Spawn vehicles at vehicle generators
+	auto camera2D = glm::vec2(camera.position);
 	for (auto& gen : world->state->vehicleGenerators) {
-		float dist2 = glm::distance2(camera.position, gen.position);
+		auto gen2D = glm::vec2(gen.position);
```

Comment:
Unless you know this for certain, add a note (`/// @todo`) that we are not sure how the check is done yet.

(Maybe ground Z is detected first and then a 3D distance is calculated? It probably doesn't matter much for the stock game, but we might have mods at some point. Also some people might be interested in figuring this stuff out for us)